### ISSUE TITLE
Measure row count effect on hot read path

### ISSUE THREAD

**Issue by jasobrown-rs** [readysettech/readyset#371]
Revive //benchmarks read tests to understand how the row of rows in a result set affects the throughput and latency of reads.

**Comment by jasobrown-rs** [comment]
Implemented benchmarking via sysbench, and wrote up a [summary/recommendation doc](https://docs.google.com/document/d/1UmgIzCNOW-y1p_7UMZHziUWxiuuFdYXzzrWCU14eGDo/edit). Marking this ticket as done.

**Comment by jasobrown-rs** [comment]
mysql performance measures correctly, or at least passes the basic sniff test of appearing correct. PG, on the other hand, is completely out to lunch, and I'm trying to figure out why.

**Comment by jasobrown-rs** [comment]
Hit a problem where even though I configured multiple row to be inserted into a table as part of the data generation phase, only one was ever being returned when executing `select * from table`. This affected both pg and mysql. Aspen and I narrowed it down to a bug in the adapter where it was incorrectly handling multi-row insert statements. Aspen [pached](https://gerrit.readyset.name/c/readyset/+/5294) this bug, and it's now correctly returning all the rows that we attempted to insert.
